lib/galaxy/schema/schema.py

```diff
@@ -3792,6 +3792,15 @@ class CreatePagePayload(PageSummaryBase):
     model_config = ConfigDict(use_enum_values=True, extra="allow")
 
 
+class UpdatePagePayload(PageSummaryBase):
+    annotation: Optional[str] = Field(
+        default=None,
+        title="Annotation",
+        description="Annotation that will be attached to the page.",
+    )
+    model_config = ConfigDict(use_enum_values=True, extra="allow")
```

Comment:
Are you using enums and do you have extra attributes that are not part of the model ? If not please remove this.